package.json

```
@@ -0,0 +1,47 @@
+{
+  "name": "return-of-pokedex",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "serve": "vue-cli-service serve",
+    "build": "vue-cli-service build",
+    "lint": "vue-cli-service lint",
```

```suggestion
    "lint": "vue-cli-service lint --no-fix",
```
Don't forget for lab requirements to put this in during BC2